### DISCUSSION TITLE
Hidden digit

### DISCUSSION THREAD

**Discussion by i-B4se** [jomjol/AI-on-the-edge-device#3234] [General]
Hello,
I've been using this for a long time and it works perfectly, but we replaced the meter last year and one of the numbers is covered by a strange design-related part, meaning it can't recognize the number.
Can you set a number at a certain point so that it only registers the others?
![image](https://github.com/user-attachments/assets/d63b9925-a87a-4a5a-b9b6-d54ed4f6bc07)
Thank you in advance

**Comment by SybexX** [comment]
It looks like you can twist this needle and it only serves to remove the bubbles or push them to an area where they won't bother you.
